pkg/dbal/DbalSubscriptionConsumer.php

```diff
@@ -36,6 +36,13 @@ class DbalSubscriptionConsumer implements SubscriptionConsumer
      */
     private $redeliveryDelay;
 
+    /**
+     * Time to wait between subscription requests in milliseconds.
+     *
+     * @var int
+     */
+    private $subscriptionInterval = 200;
```

Comment:
What about naming it pollingInterval like [here](https://github.com/php-enqueue/enqueue-dev/blob/master/pkg/fs/FsConsumer.php#L39) to stay consistent with other transports ? 

The tests should be fixed